modules/contact/src/actions/ContactAction.C

```diff
@@ -181,8 +205,8 @@ ContactAction::ContactAction(const InputParameters & params)
     if (_mortar_approach == MortarApproach::Legacy)
     {
       mooseDeprecated(
-          "Use of legacy mortar contact approach is deprecated and will be removed by December "
-          "2021. Instead, select the default option based on weighted quantities and dual bases");
+          "Use of legacy mortar contact approach is deprecated and will be removed by September "
+          "2022. Instead, select the default option based on weighted quantities and dual bases");
```

Comment:
😆 I occasionally run across deprecation messages that I put in that said I would remove something a long time ago ... and clearly haven't

Reply:
I am planning to do this by September, just postpone it over and over given that there are other more pressing items to tackle, but definitely plan to do it!